src/Stripe.Tests.XUnit/accounts/_fixture.cs

```diff
@@ -44,7 +44,7 @@ public account_fixture()
                         new StripeAccountAdditionalOwner
                         {
                             FirstName = "Bumble", LastName = "B",
-                            BirthDay = 29, BirthMonth = 8, BirthYear = 2013,
+                            BirthDay = 29, BirthMonth = 8, BirthYear = 1980,
```

Comment:
I guess this was broken by the age check that Security recently put in? Might be worth adding a comment to the effect that the `BirthYear` is somewhat important.

Reply:
I'm happy to put a comment though it feels self explanatory that the DOB should be a valid one and not for a 4 year old. Let me know if it's worth the fix though

Reply:
Alright, fair enough.